src/components/Grants/GrantCard/VestingProgress.tsx

```diff
@@ -49,7 +49,7 @@ const VestingProgress = ({ grant }: Props) => {
       </div>
 
       <div className="VestingProgressBar">
-        {releasedPercentage && releasedPercentage > 0 && (
+        {!!(releasedPercentage && releasedPercentage > 0) && (
```

Comment:
This one was funny,

 `(releasedPercentage && releasedPercentage > 0)` evaluates to a number when releasedPercentage is 0.
 
 
![image](https://user-images.githubusercontent.com/2858950/182786612-37f2d36a-ca51-4e74-804f-aa437ec7fda4.png)